test/test_cuda.py

```diff
@@ -3504,6 +3506,62 @@ def throws_on_cuda_event(capture_error_mode):
         # Exception would Corrupt Process and make other tests fail
         # self.assertTrue(throws_on_cuda_event("global"))
 
+    @unittest.skipIf(
+        not TEST_CUDA_GRAPH or not TEST_CUDA_PYTHON,
+        "CUDA >= 11.0 or ROCM >= 5.3 required for graphs, cuda-python must be installed",
+    )
+    @parametrize("instantiate_eagerly", [False, True])
+    def test_cuda_graph_raw_graph(self, instantiate_eagerly):
+        import cuda.bindings.runtime as cudart
+
+        graph = torch.cuda.CUDAGraph(instantiate_eagerly=instantiate_eagerly)
+        x = torch.zeros([2000], device="cuda")
+        y = torch.ones([2000], device="cuda")
+        with torch.cuda.graph(graph, capture_error_mode="relaxed"):
+            z = x + y
+
+        cudart_cuda_graph = cudart.cudaGraph_t(init_value=graph.raw_cuda_graph())
+        _, num_nodes = cuda_python_error_check(
+            cudart.cudaGraphGetNodes(cudart_cuda_graph)
+        )
+        nodes, _ = cuda_python_error_check(
+            cudart.cudaGraphGetNodes(cudart_cuda_graph, num_nodes)
+        )
+        for node in nodes:
+            cuda_python_error_check(cudart.cudaGraphNodeGetType(node))
+
+        graph.replay()
+
+    @unittest.skipIf(
+        not TEST_CUDA_GRAPH, "CUDA >= 11.0 or ROCM >= 5.3 required for graphs"
```

Comment:
Not your responsibility, but a "while we're in here" would be to remove the CUDA >= 11.0 condition, as we haven't supported CUDA < 11.0 for a while now.

Reply:
Ehhh... I would rather make such a sweeping change in is own PR.